diff --git a/cmd/flags/flags.go b/cmd/flags/flags.go
index cf1b603..137a381 100644
--- a/cmd/flags/flags.go
+++ b/cmd/flags/flags.go
@@ -7,7 +7,7 @@ import (
 const (
 	// DefaultHomeGPhotosUploaderCLIFolder is the default .gphotos-uploader-cli home
 	// folder where to store specific configurations
-	DefaultHomeGPhotosUploaderCLIFolder = "~/.gphotos-uploader-cli"
+	DefaultHomeGPhotosUploaderCLIFolder = "/config/docker"
 )
 
 // GlobalFlags is the flags that contains the global flags
