diff --git a/config/config.go b/config/config.go
index 82966bb..794477a 100644
--- a/config/config.go
+++ b/config/config.go
@@ -28,7 +28,7 @@ func defaultSettings() *Config {
 	c.Jobs = make([]FolderUploadJob, 0)
 	job := FolderUploadJob{
 		Account:      "youremail@gmail.com",
-		SourceFolder: "~/folder/to/upload",
+		SourceFolder: "/photos",
 		MakeAlbums: MakeAlbums{
 			Enabled: true,
 			Use:     "folderName",
