diff --git a/app/oauth.go b/app/oauth.go
index 554c3d0..f157c18 100644
--- a/app/oauth.go
+++ b/app/oauth.go
@@ -90,6 +90,8 @@ func (app *App) obtainOAuthTokenFromAuthServer(ctx context.Context, oauth2Config
 			OAuth2Config:           oauth2Config,
 			LocalServerReadyChan:   ready,
 			LocalServerSuccessHTML: successPage,
+			LocalServerAddress:     "0.0.0.0",
+			LocalServerPort:        []int{29070},
 		})
 		return err
 	})
